about.html

```diff
@@ -31,6 +31,7 @@
             <i class="fas fa-bars hamb"></i>
         </nav>
     </header>
+
```

Comment:
[REQUIRED] Good job including the `see the whole programme` text but as I have stated we want your design to look like the template so kindly include the underline.